tests/sentry/integrations/gitlab/test_search.py

```diff
@@ -80,14 +104,34 @@ def test_finds_project_results(self):
     def test_finds_no_external_issues_results(self):
         responses.add(
             responses.GET,
-            'https://example.gitlab.com/api/v4/groups/1/issues?scope=all&search=XYZ',
+            'https://example.gitlab.com/api/v4/projects/5/issues?scope=all&search=XYZ',
             json=[]
         )
         resp = self.client.get(
             self.url,
             data={
                 'field': 'externalIssue',
                 'query': 'XYZ',
+                'project': '5',
+            }
+        )
+
+        assert resp.status_code == 200
+        assert resp.data == []
+
+    @responses.activate
+    def test_finds_no_external_issues_results_iid(self):
+        responses.add(
+            responses.GET,
+            'https://example.gitlab.com/api/v4/projects/5/issues?scope=all&search=11',
+            json=[]
+        )
+        resp = self.client.get(
+            self.url,
+            data={
+                'field': 'externalIssue',
+                'query': '11',
+                'project': '5',
```

Comment:
You might want one more test case for the scenario when user manages to send a request off with no project.